feat(icons): new icons september thru november

conradennis opened the pull request:
Submitting new and updated UI icons from September through November 2023 all updates and additions approved by BXD

Note: please hide the two deprecated icons from the Carbon/IDL library

70 net new icons
1 updated icon

1 new icon category
8 icons moved to new category

2 deprecated icons

updated categories.yml with additions and updates
updated deprecated.yml with deprecated file names
updated ui-icon-master.ai with additions and updates

@LMapes and I will update friendly names and aliases in an additional PR within the next week or so


#### Changelog

**New** 

– 32/AI.svg
– 32/ai-governance--lifecycle.svg
– 32/ai-governance--tracked.svg
– 32/ai-governance--untracked.svg
– 32/arithmetic-mean.svg
– 32/arithmetic-median.svg
– 32/async.svg
– 32/calls.svg
– 32/calls--all.svg
– 32/calls--incoming.svg
– 32/chart--planning-waterfall.svg
– 32/chevron--down--outline.svg
– 32/chevron--up--outline.svg
– 32/customer.svg
– 32/data-analytics.svg
– 32/database--messaging.svg
– 32/demo.svg
– 32/devices--apps.svg
– 32/downstream.svg
– 32/equal--approximately.svg
– 32/event--change.svg
– 32/event--incident.svg
– 32/event--warning.svg
– 32/foundation-model.svg
– 32/ibm--data-product-exchange.svg
– 32/ibm--dynamic-route-server.svg
– 32/ibm--engineering-systems-design-rhapsody-model-manager.svg
– 32/ibm--engineering-systems-design-rhapsody-sn1.svg
– 32/ibm--engineering-systems-design-rhapsody-sn2.svg
– 32/ibm--ibv.svg
– 32/ibm--instana.svg
– 32/ibm-watsonx--code-assistant-for-z--refactor.svg
– 32/ibm-z--processor-capacity-reference.svg
– 32/infinity.svg
– 32/IP.svg
– 32/js-error.svg
– 32/kubernetes--ip-address.svg
– 32/kubernetes--pod.svg
– 32/logo--x.svg
– 32/MAC.svg
– 32/mobile--crash.svg
– 32/mobile--event.svg
– 32/mobile--request.svg
– 32/mobile--session.svg
– 32/mobile--view-orientation.svg
– 32/mobile--view.svg
– 32/mysql.svg
– 32/network-interface.svg
– 32/platforms.svg
– 32/process.svg
– 32/process-automate.svg
– 32/prompt-template.svg
– 32/running.svg
– 32/sales-ops.svg
– 32/service-levels.svg
– 32/subdirectory.svg
– 32/subtract--filled.svg
– 32/time--filled.svg
– 32/tour.svg
– 32/upstream.svg
– 32/URL.svg
– 32/video-player.svg
– 32/warning--diamond.svg
– 32/warning--multiple.svg
– 32/web-services--cluster.svg
– 32/web-services--container.svg
– 32/web-services--definition.svg
– 32/web-services--service.svg
– 32/web-services--task-definition-version.svg
– 32/web-services--task.svg


**Updated**

– 32/ibm-cloud--transit-gateway.svg

**Moved categories**

– 32/machine-learning.svg
– 32/machine-learning-model.svg
– 32/model--tuned.svg
– 32/prompt-session.svg
– 32/tuning.svg
– 32/watsonx-ai.svg
– 32/watsonx-data.svg
– 32/watsonx-governance.svg

**Deprecated**

– 32/watson--machine-learning.svg
– 32/watson.svg

conradennis committed:
feat(icons): new icons september thru november

Submitting new and updated UI icons from September through November 2023
all updates and additions approved by BXD

Note: please hide the two deprecated icons from the Carbon/IDL library

[70] net new icons
[1] updated icons
[8] icons moved to new category
-
[1] new icon category
-
[2] deprecated icons

updated categories.yml with additions and updates
updated deprecated.yml with deprecated file names
updated ui-icon-master.ai with additions and updates

@lmapes to update friendly names and aliases in an additional PR

tw15egan committed:
chore(test): update snapshots

tw15egan committed:
Merge branch 'main' into icons-new-sept-nov-2023

tay1orjones committed:
Merge branch 'main' into icons-new-sept-nov-2023

tay1orjones committed:
chore: update snapshots

laurenmrice commented on e2e/icons-react/__snapshots__/PublicAPI-test.js.snap:
```diff
@@ -18,6 +18,10 @@ Array [
   "AddFilled",
   "AddLarge",
   "AgricultureAnalytics",
+  "Ai",
```
This may be a dumb question, but does it matter that this says "Ai" with a lowercase "i" rather than an uppercase "I" like "AI". I just noticed in other places it is referred to as "AI" for this particular svg.

conradennis replied:
For this one since it stands for artificial intelligence the "I" should probably be capitalized.

conradennis replied:
@laurenmrice see my other comment below but I changed my mind and think it's better to leave em alone

laurenmrice replied:
Okay cool!

laurenmrice commented on e2e/icons-react/__snapshots__/PublicAPI-test.js.snap:
```diff
@@ -1130,12 +1168,14 @@ Array [
   "Logout",
   "Loop",
   "LowSeverity",
+  "Mac",
```
Same with this, does it matter it isn't uppercase?

conradennis replied:
here MAC stands for Media Access Control so it should probably be capitalized

conradennis replied:
@laurenmrice actually I'm looking through some of the other icon names that are abbreviations and it looks like they all have the same treatment of first letter upper case and the rest lower case so maybe it's fine to leave them all alone

laurenmrice commented on e2e/icons-react/__snapshots__/PublicAPI-test.js.snap:
```diff
@@ -1881,6 +1940,8 @@ Array [
   "Upgrade",
   "Upload",
+  "Upstream",
+  "Url",
```
And here?

conradennis committed:
updated meta data for Carbon for Coud

changed friendly name for UI icon ibm-cloud--pal from IBM Cloud Pattern and Asset Library to Carbon for Cloud

updated aliases in icons.yml

conradennis committed:
Merge branch 'icons-new-sept-nov-2023' of https://github.com/conradennis/carbon into icons-new-sept-nov-2023

conradennis committed:
Update to icon ibm-cloud--pal artwork

Mike noticed the SVG artwork was wonky. Submitting the updated artwork.

laurenmrice reviewed (approved):
Looks good!
